applications/Unity.GrantManager/modules/Unity.Flex/src/Unity.Flex.Application.Contracts/Scoresheets/IScoresheetAppService.cs

```diff
@@ -19,6 +19,7 @@ public interface IScoresheetAppService : IApplicationService
         Task UpdateAsync(Guid scoresheetId, EditScoresheetDto dto);
         Task<List<Guid>> GetNonDeletedNumericQuestionIds(List<Guid> questionIdsToCheck);
         Task<List<QuestionDto>> GetNonDeletedYesNoQuestions(List<Guid> questionIdsToCheck);
+        Task<List<QuestionDto>> GetNonDeletedSelectListQuestions(List<Guid> questionIdsToCheck);
```

Comment:
Please look at naming here? As well as async suffixes in the contract and implementation class (double check the proxy class calls still work)
Can make some of these method private that is not part of the contract

Reply:
hi @AndreGAot - done changing the function names.